### DISCUSSION TITLE
本番向けにまとめるドキュメントについて

### DISCUSSION THREAD

**Discussion by TakumaNakagame** [cloudnativedaysjp/dreamkast-broadcast#250] [General]
必要なものを洗い出す

**Comment by TakumaNakagame** [comment]
ドキュメントをまとめているものはなし

**Comment by TakumaNakagame** [comment]
運用マニュアル（PK1）、StreamYardのオペレーションマニュアル（前回作ったやつ）はある

**Reply by tsukaman** [reply]
- P1さん作成の [運営マニュアル](https://drive.google.com/file/d/1dvfa7lKbFA1Ka8iHSUYxJSNRLNXxpHEp/view?usp=sharing)

**Reply by tsukaman** [reply]
- [スケジュール表](https://docs.google.com/spreadsheets/d/1fR1J2q8-VWLHH7bD-JGmdYokNnyrTQUHudna_XwC4F8/edit#gid=1456186091)

**Reply by tsukaman** [reply]
- [CNDT2022_PreEvent_SpeakerGuide](https://docs.google.com/presentation/d/1JI7MIYY2jKA-XERgmwfdjmXVmamM868adIufhAiQBq0/edit?usp=share_link)
- [CNDT2022_PreEvent_OperatorGuide](https://docs.google.com/presentation/d/1cjRw55KuoIV6F2G4F8NeRbwCYLaTUsFj--OzBQ8bV_s/edit?usp=sharing)

**Reply by tsukaman** [reply]
SpeakerGuideとOperatorGuideは @tsukaman でアップデートする

**Reply by tsukaman** [reply]
Updated

- [CNDT2022_SpeakerGuide（オンライン登壇の手引き）](https://docs.google.com/presentation/d/1iGTwTmTBrO-zsmUt4hMcGUFzKQj2B-prOiUvc4QHDmA/edit#slide=id.g177ece08c12_0_147)
- [CNDT2022_OperatorGuide](https://docs.google.com/presentation/d/16AdMUZaVHherA5R3H7Y6Q-yWCdt7ILLEZ3ZskrCBffQ/edit#slide=id.g177ece08c12_0_147)

**Comment by TakumaNakagame** [comment]
関係ありそうな
- https://github.com/cloudnativedaysjp/broadcast/issues/213
- https://github.com/cloudnativedaysjp/broadcast/issues/234
- https://github.com/cloudnativedaysjp/broadcast/issues/248

**Comment by TakumaNakagame** [comment]
作る必要がありそうなドキュメントリスト

## 優先度必須
- [ ] 現地登壇オペレーション（ブイキューブ向け）
    - プログラムシートのA-Dのみ抽出
    - 現地登壇の流れを書く
    - それぞれの動きは @KaseiKondo さんが持っていそう（もしくは決められる）
    - @TakumaNakagame , @KaseiKondo 
- [ ] 人員配置
    - [ ] @TakumaNakagame 
- [ ] トラブル発生時対応
    - [ ] トラブル判断者
        - [ ] @nabemasat 
        - [ ] @KaseiKondo 
    - [ ] アナウンス系の流れ
        - [ ] @nabemasat 
    - [ ] 操作系の流れ
        - [ ] dkのdelay入力
        - [ ] @TakumaNakagame 
- [ ] バックアッププラン
    - [ ] 内容
    - [ ] 切り替え方法
    - [ ] @TakumaNakagame
- [ ] PK1スタッフ向けに、環境構成図を作る
    - [ ] @TakumaNakagame 
- [x] 全体進行表
    - [x] オンライン登壇
         - cnsecのときをそのまま流用する
         - 4人しかいないし
         - つくる人： @tsukaman

## なくてもなんとかなる

- [ ] 当日のEMTECスイッチングオペレーション
    - 対象： クラウドのOBSを操作するメンバー
    - セッション内容と操作が異なるケースがあるので、これらを洗い出す
    - スケジュール表をベースとして作る
    - つくる人： @TakumaNakagame 
- [ ] スタッフガイド
    - @nabemasat さんが作ってるらしい（作ってない…？
    - 忘れ物とか
    - パーカー来てください
    - マスクしてね
    - 担当者一覧
    - 連絡先
    - ヘッドセット点けてる人一覧

**Comment by TakumaNakagame** [comment]
大量に印刷しなきゃいけないので、誰宛にいつまでに送るか
